changelog.json

```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "1.4.3-beta1": [
+      "[Added] Add Remove Repository keyboard shortcut - #5848. Thanks @say25!",
+      "[Fixed] Fix intermittent emoji autocompletion loading issue - #5859",
+      "[Fixed] Correct Publish branch modal window's default button - #5777. Thanks @JKirkYuan!",
+      "[Fixed] Support python3 in GitHub Desktop CLI - #5843. Thanks @munir131!"
```

Comment:
:art: Suggested copy tweak (make it past tense, avoid unnecessary Product Names™, indicate it's a platform-specific fix):

> `[Fixed] Desktop CLI does not work with python 3 on macOS ...`

Reply:
Also, Python is a proper noun since this isn’t referring to the [actual snakes](https://en.wikipedia.org/wiki/Pythonidae) 🐍.

Reply:
![rotating python skull](https://user-images.githubusercontent.com/964912/46880515-ad236c80-cdfd-11e8-883c-acbfd4870a84.gif)